certbot/renewal.py

```diff
@@ -190,6 +191,15 @@ def _restore_required_config_elements(config, renewalparams):
                     raise errors.Error(
                         "Expected a numeric value for {0}".format(config_item))
             setattr(config.namespace, config_item, int_value)
+    # bool-valued items to add if they're present
```

Comment:
Sorry, I currently can't even test if it runs without a unit test ;-) Nope, I can't do that without learning a lot of python specific stuff. Which I currently don't want to.